ui/app/styles/components/tabs.scss

```diff
@@ -11,7 +11,9 @@
   box-shadow: inset 0 -1px 0 $grey-light;
 
   ul {
-    border-color: transparent;
+    align-items: center;
```

Comment:
thanks for putting the properties in alphabetical order! makes it easier to find + read!

Reply:
Are we planning to keep properties alphabetized in the future? If so, we should add it to the patterns doc!

Reply:
Good point. I started doing it for my own organization (to help compare against the current CSS). But I'll add it to the UI sync topic list.